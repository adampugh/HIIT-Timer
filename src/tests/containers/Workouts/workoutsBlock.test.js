import React from "react";
import { shallow } from "enzyme"; 

import WorkoutsBlock from "../../../containers/Workouts/workoutsBlock";
import workoutState from "../../fixtures/workouts";

test("should render editBlock correctly", () => {
    const workout = workoutState.workouts[0];
    const wrapper = shallow(<WorkoutsBlock 
        title={workout.title} 
        totalTime={workout.totalTime} />);
    expect(wrapper).toMatchSnapshot();
});