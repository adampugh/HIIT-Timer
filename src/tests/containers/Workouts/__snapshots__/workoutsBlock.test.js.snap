// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render editBlock correctly 1`] = `
<div
  className="workoutsBlock"
>
  <h1>
    Leg Workout
  </h1>
  <h3>
    60 Minutes
  </h3>
  <i
    className="far fa-play-circle"
  />
</div>
`;
